tiptapy/__init__.py

```diff
@@ -69,6 +69,14 @@ def inner_render(self, node) -> str:
         return html
 
 
+class Embed(BaseContainer):
+    type = "embed"
+
+    def inner_render(self, node):
+        attrs = node['attrs']
+        return attrs.get('html')
```

Comment:
@shon I have updated from 
```python
# attrs.get('html', []) -> attrs.get('html')
```
Now it would just return `None` if the it `html` key is not present in the JSON.

Reply:
Having a word with @deepaksisodiya I was told that all the HTML would be generated at the backend and I would just need to return the HTML as it is for the `embed` block 😄

Reply:
Yes, but if there is no **html** key, you should also consider that condition 😄